profiles/common/themes/ec_resp/css/interinstitutional_less/hack-header.less

```diff
@@ -75,6 +75,8 @@
   .easy-breadcrumb_segment-separator {
     color: #82B0BE;
     font-weight: bold;
+    background:url("../images/breadcrumb.gif") no-repeat scroll 2px 0px;
```

Comment:
Coding standards: Missing space after the colon. See https://www.drupal.org/node/1887862

> The property name should be immediately followed by a colon, then a single space, and then the property’s value.